### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Accessible environment_Desired Storage Explorer theme_ColorContrast:"Explorer" text present in the Explorer Pane is having Color contrast less that 4.5:1

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/AzureStorageExplorer#5732]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHub Tags:**
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev2; #BM_Azure Tools Storage Explorer_May2022; #Azure Tools Storage Explorer; #WCAG1.4.3; #FTP;  #ColorContrast; #Regressed:06-07-22

## Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.24.0

## OS Details
Microsoft Windows 11 Enterprise
version 21H2 OS Build 22000.613

## Repro Steps:

1. Launch Storage Explorer
2. Navigate to "View" menu and click on it.
3. Navigate to "Themes" and click on it
4. Select System Default/Light theme.
5. Use Color Contrast Tool to check the color contrast ratio for "Explorer" text present in the "Explorer" pane.

## Actual results:
"Explorer" text has color contrast of 4.499:1 which is less than 4.5:1 in System Default/Light Theme.

**Similar issues observed in the following:**
In Dark Theme
"Explorer" text has CC of 4.494:1 which is less than 4.5:1.

## Expected results:
Text should have minimum color contrast ratio of 4.5:1 

## User Impact:
Low vision Users might fail to acknowledge the element if it doesn't have minimum color contrast ratio.

## Attachments:
![5732_A11y_AzureToolsStorage_DefaultOrLightTheme](https://user-images.githubusercontent.com/105198219/168581898-529216d8-59ca-49c1-bc93-123a63d92262.png)
![5732_SimilarIssue_DarkTheme_ExplorerText](https://user-images.githubusercontent.com/105198219/168582236-7c9c0f23-3e0f-45c4-b6f5-12fffb2b2778.png)

**Comment by ManishaC1** [comment]
Please re-open this bug as the issue is still repro'ing in the latest environment version:1.24.2. 
"Explorer" text has color contrast of 4.499:1 which is less than 4.5:1 in System Default/Light Theme.
"Explorer" text has CC of 4.494:1 which is less than 4.5:1 in Dark Theme.

**Comment by craxal** [comment]
@ManishaC1 Can you please share the build number of the build you last used to verify this? We normally close bugs when a merge has been made to our master branch and is available in our internal daily builds, which we don't usually distribute.

**Comment by ManishaC1** [comment]
@craxal I have updated the application to the latest version v1.24.2 Build number:20220527.5. Don't know how to get the earlier version and build number.

**Comment by craxal** [comment]
Yeah, that's doesn't contain the fix. However, I'll keep the issue open and assign to Verifying until you get a build containing the fix and can verify it's resolved.

**Comment by MRayermannMSFT** [comment]
Hi @ManishaC1 

Please verify that this bug is fixed using the Storage Explorer build at: https://storageexplorerpublish.blob.core.windows.net/privatebuilds/20220624.3-accessibility-testing/StorageExplorer-ia32.exe

**Comment by ManishaC1** [comment]
@MRayermannMSFT 
CC for "explorer text" in system default theme is having specified minimum Color contrast but failing in dark theme. Attachments are provided below.
![5732_Issue not repro](https://user-images.githubusercontent.com/105198219/175873357-97f8c8a6-284a-4224-9253-b613c070cf2d.png)

Below attachment shows the "explorer" text in dark them falls below minimum specified CC ratio i.e., 4:5:1
![5732_Issue repro](https://user-images.githubusercontent.com/105198219/175873367-bd5a6cd6-f0ec-4c48-9338-202b65385f33.png)

**Comment by craxal** [comment]
@ManishaC1 Sorry, this was unassigned from me, and it fell off my radar. Please verify that this is fixed using [this new build](https://cralvordtest02.blob.core.windows.net/customer-share/internal/accessibility-1.25.0.1/StorageExplorer-ia32.exe?sv=2021-04-10&st=2022-07-14T21%3A02%3A15Z&se=2022-07-23T23%3A59%3A00Z&sr=b&sp=r&sig=SFQBoacwmcjSUAl0zibeOo1CZp6XaRLb9Dm7I9Xondk%3D).

**Comment by ManishaC1** [comment]
The issue is resolved in the provided environment. Hence closing the bug. Please refer the attachment.
![Screenshot (535)](https://user-images.githubusercontent.com/105198219/179154067-59e53074-a2c3-4a16-bc2d-4a7a0645aee7.png)

**Comment by ManishaC1** [comment]
#Github tags:#Closed

**Comment by Vishnuvarthancr7** [comment]
GitHubTags:#Closed;
